eucaconsole

```diff
@@ -35,18 +36,10 @@ def daemonize(func=None):
     if procid == -1:
         sys.exit(1)
 
-    # Close open file descriptors
-    for fd in (0, 1, 2):
-        try:
-            os.close(fd)
-        except OSError:
-            pass
-
-    print("about to point fds to null")
-    # Point descriptors to /dev/null
-    os.open("/dev/null", os.O_RDWR)
-    os.dup(0)
-    os.dup(0)
+    # output descriptors point to log file
+    f = os.open(log_file, os.O_WRONLY|os.O_APPEND)
+    os.dup2(f, 1)
+    os.dup2(f, 2)
```

Comment:
Shouldn't this still be closing stdin?  (I'd assume the high-level functions are fine for that.)